RULES.md

```diff
@@ -0,0 +1,101 @@
+Those areas are going to be segmented by
+
+- Products (RHEL, Ubuntu, …) - make it easy to develop content of a product that doesn’t influence the other content or the build system.
+- Product-specific profiles (PSPs s.a. Firefox STIG, Ubuntu CIS, …) and respective control files - make it easy for SMEs that only want to assign rules to profiles without going into details.
+- Shared resources
+  - Product-independent profiles (PIPs s.a. ANSSI, HIPAA, PCI-DSS, …) and respective control files - ensure that PIP development benefits the whole community instead of cluttering the content with “if product in [...]”.
+  - Build system - decisions upon architecture of the build system impact build time, project capabilities, and can also move maintenance costs.
+  - Test-related code - ensure that tests have the greatest coverage as possible, but that don’t waste time and don’t suffer from false positives.
+  - Other - rules only loosely coupled to products, templates, CPEs etc. Same principles that apply to PIPs apply for the shared content as well.
+- Organization administration - ability to grant and remove rights, create and delete repositories etc.
+
+
+## Implementation
+
+### Products
+
+- Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
+- Area of effect: All PSPs that don’t have specific stakeholders, associated content and test scenarios.
+- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy paste, and also don’t change existing behavior for everybody (link to coding guidelines needed).
+
+
+### Product Specific Profiles (PSPs)
+
+- Keeping track of the decomposition: Metadata in profile yml files, CODEOWNERS file
+- Area of effect: Respective profile files
+- Guidelines: N/A
+
+
+### Product Independent Profiles (PIPs)
+
+- Keeping track of the decomposition: Metadata in control files. Any profile definitions that live as separate profile files are simply a product’s content.
+- Area of effect: Just control files
+- Guidelines: Let’s try and see whether the control file format helps us with effort sharing.
+
+
+### Build system
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Everything that is directly or indirectly used during the content build process
+- Guidelines: We need strong protection against technical debt, and a framework to deal with deprecations or changes of behavior.
+
+
+### Shared Resources
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Content that impacts more products.
+- Guidelines: The same as for the build system.
+
+
+### Tests
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Test scenarios, test suite features, unit tests, gating tests
+- Guidelines: We need guidelines to preserve test robustness and the rate of tests usefulness.
```

Comment:
I would suggest to explicitly inform the intention for continuous performance improvements. Also valid for `Build system".